templates/components/password-reset-expired.jsx

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var IconText = require('./icontext.jsx');
+
+var PasswordResetExpired = React.createClass({
+  render: function() {
+    return (
+      <IconText
+        className="messageBox"
+        iconClass="messageIcon fa fa-exclamation-triangle"
+        header="Seems that the link in your email expired."
+        headerClass="messageHeader">
+          <p>
+            Request a new reset password link below
```

Comment:
perhaps "Request a new reset link for setting a new password below", to mitigate possible confusion around saying "request a link" here, but "set a password" on the actual on-page button? (see https://github.com/mozilla/id.webmaker.org/issues/360#issuecomment-134677659)